index.html

```diff
@@ -5,9 +5,40 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
-    <title>Document</title>
+    <title>Portfolio</title>
 </head>
 <body>
-    <h1>hello</h1>
+    <div id="header-background-mobile">
+      <header id="Toolbar-Dark-Compact">
+         <div id="Logo-Placeholder">
+             <p id="My-Logo">Qwerty</p>
+         </div>
+         <div id="Normal-Button-Tertiary-Enabled">         
+            <div id="icon-menu"><img src="img/NormalButton/Tertiary/Icons/Union.png" alt="Menu"></div>
+         </div>
+      </header>
+      <div id="Headline">
+         <h2 id="Primary-Text">I’m Danys
+             Glad to see you!
+         </h2>
+         <p id="Primary-Content">I’m a software developer! I can help you build
+             a product , feature or website Look through some
+             of my work and experience! If you like what you
+             see and have a project you need coded,
+             don’t hestiate to contact me.
+          </p>          
+          <div id="Social-Media">
+              <p id="Lets-Connect">LET’S CONNECT</p>            
+          </div>
+          <ul id="social-media-icons">
+            <li><a href="https://twitter.com/Danys_Linares" target="_blank" rel="noopener noreferer"><img src="img/Vector-twitter.svg" alt="Twitter link"></a></li>
+            <li><a href="https://www.linkedin.com/in/danys-linares-6a328b238?lipi=urn%3Ali%3Apage%3Ad_flagship3_profile_view_base_contact_details%3BnkyI5IMjTzSg4PVJIZh%2BMw%3D%3D" rel="noopener noreferer"><img src="img/Linkedin-icon.svg" alt="Linkedin link"></a></li>
+            <li><a href="https://medium.com/" target="_blank" rel="noopener noreferer" ><img src="img/Vector-M.svg" alt="Medium link"></a></li>
+            <li><a href="https://github.com/d4nQw3rty" target="_blank" rel="noopener noreferrer"><img src="img/Vector-github.svg" alt="Github link"></a></li>
+            <li><a href="https://www.angellist.com/" target="_blank" rel="noopener noreferrer"><img src="img/Vector-hand.svg" alt="angellist link"></a></li>
+         </ul>                                  
+      </div>
+    </div>
```

Comment:
- [ ] Kindly change your `div` tag to a `section` tag wrapping the whole section, the element represents a generic standalone section of a document, learn more[ here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section)